packages/web/docs/src/pages/docs/features/app-deployments.mdx

```diff
@@ -0,0 +1,371 @@
+  plugins: [
+    useHive({
+      experimental__persistedDocuments: {
+        cdn: {
+          // replace <target_id> and <cdn_access_token> with your values
+          endpoint: 'https://cdn.graphql-hive.com/<target_id>',
+          accessToken: '<cdn_access_token>'
+        }
+      }
+    })
+  ]
+})
+```
+
+For further configuration options, please refer to the
+[Hive Client API reference](/docs/api-reference/client#persisted-documents).
+
+</Tabs.Tab>
+
+{/* GraphQL Mesh */}
+
+<Tabs.Tab>
+
+For GraphQL Mesh you can use the Hive configuration for resolving persisted documents. Adjust your
+`mesh.config.ts` file as follows.
+
+```typescript filename="mesh.config.ts" {7-13}
+import { defineConfig } from '@graphql-mesh/serve-cli'
+
+export default defineConfig({
+  hive: {
+    // The endpoint of CDN
+    endpoint: 'https://cdn.graphql-hive.com/<target_id>/supergraph',
+    experimental__persistedDocuments: {
+      cdn: {
+        // replace <target_id> and <cdn_access_token> with your values
+        endpoint: 'https://cdn.graphql-hive.com/<target_id>',
+        accessToken: '<cdn_access_token>'
+      }
+    }
+  }
+})
+```
+
+For further configuration options, please refer to the
+[Hive Client API reference](/docs/api-reference/client).
+
+</Tabs.Tab>
+
+{/* Custom Third Party */}
+
+<Tabs.Tab>
+
+Any published app deployment is available via the Hive CDN. You can fetch a the persisted documents
+by doing a HTTP GET request.
+
+```text filename="Fetch Persisted Document via HTTP"
+GET https://cdn.graphql-hive.com/<target_id>/apps/<app_name>/<app_version>/<document_hash>
+```
+
+| Parameter       | Description                                                             |
+| --------------- | ----------------------------------------------------------------------- |
+| `target_id`     | The target ID of the Hive target where the app deployment is published. |
+| `app_name`      | The name of the app deployment.                                         |
+| `app_version`   | The version of the app deployment.                                      |
+| `document_hash` | The hash of the persisted document.                                     |
+
+**Example curl request:**
+
+```bash filename="Example curl request"
+curl \
+  -L \
+  -H 'X-Hive-CDN-Key: CDN_ACCESS_TOKEN' \
+  https://cdn.graphql-hive.com/<target_id>/apps/<app_name>/<app_version>/<document_hash>
+```
+
+**Expected responses:**
+
+| Response Status | Description                                                             |
+| --------------- | ----------------------------------------------------------------------- |
+| 200             | The persisted document was found and is returned in the response body.  |
+| 404             | The persisted document does not exist.                                  |
+| 401             | The request is unauthorized. A valid CDN access token must be provided. |
+
+</Tabs.Tab>
+
+</Tabs>
+
+## Sending Persisted Document Requests from your App
```

Comment:
I think we should add docs for GraphQL Client specific integrations. Similar to what is being done for integration on servers we should have for clients

Reply:
agreed, lets do it in a follow-up PR